api/orchestrator/orchestrator.go

```diff
@@ -33,6 +33,7 @@ import (
 )
 
 type CloudServiceHookFunc func(ctx context.Context, cld *CloudService, err error)
+type ToeHookFunc func(ctx context.Context, toe *TargetOfEvaluation, status ToeStatus_Status, err error)
```

Comment:
I think we only used "Toe" in the URLs, in the code itself I would use the "long" version TargetOfEvaluation.